gitrevise/odb.py

```diff
@@ -495,6 +547,8 @@ def _parse_body(self):
                 self.author = Signature(value)
             elif key == b"committer":
                 self.committer = Signature(value)
+            elif key == b"gpgsig":
+                self.gpgsig = value
```

Comment:
nit: please initialize `self.gpgsig` to `None` outside of this block, so it always has avalue.